hangups/http_utils.py

```diff
@@ -46,6 +46,9 @@ def fetch(method, url, params=None, headers=None, cookies=None, data=None,
         else:
             error_msg = None
             break
+        finally:
+            if 'res' in locals():
+                res.close()
```

Comment:
We are not using the 'safe reading' method ```aiohttp.ClientResponse.read```, we read directly from the stream.
And even if so, this would only cover errors (client/server communication related, ```asyncio.CancelledError``` or lowlevel in aiohttp) occuring while reading from the stream.

I see exits in the following lines, which result in an unclosed response:
* 316, 318: intended
* 328, 330, 333: while reading from stream
* 340: ```asyncio.CancelledError``` or unexpected error during handling the received content